composer.json

```diff
@@ -90,7 +90,8 @@
     },
     "patches": {
       "phpunit/phpunit-mock-objects": {
-        "Fix ReflectionParameter warnings on PHP 8": "patches/phpunit-mock-objects.patch"
+        "Fix ReflectionParameter warnings on PHP 8": "patches/phpunit-mock-objects.patch",
```

Comment:
Did this patch just need to be refreshed?

Reply:
Or rather, I'm curious how you came up with this change: https://github.com/ampproject/amp-wp/pull/7088/commits/4725407626b0d28e3866a6e692dc07cb200cb592

Reply:
Yes, We need to update that patch. Because previously it was used the ^3.2 version. and now it's using the ^5.0.9 version. But after [3.4.4](https://github.com/sebastianbergmann/phpunit-mock-objects/tree/3.4.4) directory structure is changed ( see [4.0.0](https://github.com/sebastianbergmann/phpunit-mock-objects/tree/4.0.0) ) So therefor we need to update patch file.